opae.spec.fedora

```diff
@@ -13,11 +13,11 @@ Requires:       uuid
 Requires:	json-c
 Requires:	python3
 Requires:	systemd
-Requires:	libcap
 Requires:	tbb
 Requires:	hwloc
 Requires:	spdlog
-Requires:	libedit
+Requires:	cap
+Requires:	edit
```

Comment:
We still want libedit here.  That's the name of the package:  https://src.fedoraproject.org/rpms/libedit
cap can be removed, as we will no longer be building fpgaperf_counter.

Reply:
added libedit  to Requires tag